build.gradle

```diff
@@ -45,21 +45,21 @@ def ensureBuildPrerequisites(rbuildPrerequisitesMessage) {
     if (!JavaVersion.current().isJava8Compatible()) {
         throw new GradleException(
                 "Java 8 or later is required to build Picard, but ${JavaVersion.current()} was found. "
-                        + "$buildPrerequisitesMessage")
+                        + "$rbuildPrerequisitesMessage")
```

Comment:
Thanks for taking the time to look at this. I think the right to fix this is to change the argument to `buildPrerequisitesMessage`, which I believe is what @lbergelson meant to type in the first place.
```
def ensureBuildPrerequisites(buildPrerequisitesMessage) {
```